book/push/push-ios.md

```diff
@@ -0,0 +1,90 @@
+
+# Push plugin - iOS
+
+The iOS push plugin for Zapp is based on implementing the `ZPPushProviderProtocol`.
+The protocol goes through all of the functions for initializing a push provider.
```

Comment:
either `this protocol` or `the ZPPushProviderProtocol`

Reply:
Done